lib/diagnostic.rb

```diff
@@ -108,10 +108,13 @@
 # Use shorthand to make this one line long.
 # Write your response as a Ruby string.
 
-Diagnostic.person_hash = 'your answer here'
+Diagnostic.person_hash = 'class Person attr_accessor :favorite_number
+attr_accessor :first_name def initialize (favorite_number, first_name)
+@favorite_number = favorite_number @first_name = first_name end end'
+# "van = Person.new(28, 'Van')"
```

Comment:
We were looking for the short hand version of creating a hash:

`'person = {first_name: "Van", favorite_number: 28}'`